administering_a_cluster/dedicated-admin-role.adoc

```diff
@@ -1,7 +1,8 @@
 [id="dedicated-administrator-role"]
 = The dedicated-admin role
 include::modules/common-attributes.adoc[]
-:context: dedicated-administrator
+:context: dedicated-administrator-role
```

Comment:
dedicated-admin-role?

Reply:
Good catch, I missed that the file name was different too. Will update

Reply:
Yes, there is a dedicated-admin and cluster-admin roles in OSD/ROSA/ARO.